deltachat-ios/View/InitialsBadge.swift

```diff
@@ -90,11 +90,14 @@ class InitialsBadge: UIView {
         label.font = font
     }
 
-    func setImage(_ image: UIImage) {
+    func setImage(_ image: UIImage, _ accessibilityLabelText: String? = nil) {
         self.imageView.image = image
         self.imageView.contentMode = UIView.ContentMode.scaleAspectFill
         self.imageView.isHidden = false
         self.label.isHidden = true
+        if let text = accessibilityLabelText {
+            accessibilityLabel = "avatar \(text)"
```

Comment:
if this function is only called by https://github.com/deltachat/deltachat-ios/pull/663/files#diff-e140c04537dd6b083c550047ebd58506R277 i would not prepend "avatar" here